packages/ui/build/vite/build.mjs

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env zx
+
+import { $ } from 'zx'
+import fs, {
+  rmSync,
+  statSync,
+  existsSync,
+  renameSync,
+  unlinkSync,
+  readdirSync,
+  appendFileSync,
+} from 'fs'
+import {
+  readDirRecursive,
+  kebabToPascalCase,
+  getSplitDataByPath,
+  deleteHashFromFilename,
+} from './utils.mjs'
+
+/**
+ * removing dist dir before we start
+ */
+if (fs.existsSync('./dist')) {
+  fs.rmSync('./dist', { recursive: true })
+}
+
+/**
+ * making esm-node (mjs) build separately because for some reasons it has conflicts with styles build
+ */
+await $`vite build --config ./build/vite/configs/vite.mjs.js`
+
+/**
+ * parallel build for all formats
+ */
+await Promise.all([
+  $`vite build --config ./build/vite/configs/vite.cjs.js`,
+  $`vite build --config ./build/vite/configs/vite.iife.js`,
+  $`vite build --config ./build/vite/configs/vite.esm.js`,
+  $`vite build --config ./build/vite/configs/vite.styles.js`,
+  $`yarn build:types`,
+])
+
+/**
+ * renaming css files from esm-node build (deleting hash from name)
+ */
+await readDirRecursive('./dist/esm-node/src/components').forEach((filePath) => {
```

Comment:
```suggestion
/** Renaming css files from esm-node build (deleting hash from name) */
await readDirRecursive('./dist/esm-node/src/components').forEach((filePath) => {
```

This will work too.